DuckDuckGo/Preferences/View/PreferencesVPNView.swift

```diff
@@ -174,6 +203,12 @@ extension Preferences {
                 }
             }
         }
+
+        /// Resolves the text to be used for exclusion counts
+        ///
+        private func exclusionCountString(value: Int) -> String {
+            value > 0 ? String(value) : "None"
```

Comment:
Are we not translating `"None"`?

Reply:
I'll push this string change in the follow-up PR since translations are already in and this is behind a feature flag anyway, if that's fine.

Reply:
For reference, [here's the localization](https://github.com/duckduckgo/macos-browser/blob/de4e6faf0282e667fa8153846b6c2334abaced32/DuckDuckGo/Preferences/View/PreferencesVPNView.swift#L210).

Reply:
Totally fine 👍